.github/workflows/ci.yml

```
@@ -0,0 +1,23 @@
+      - "**.md"
+      - "mkdocs.yml"
+  workflow_dispatch:
+  
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
```

actionlint (1.7.8)

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)